filament/backend/src/opengl/OpenGLContext.h

```diff
@@ -429,14 +428,6 @@ void OpenGLContext::bindVertexArray(RenderPrimitive const* p) noexcept {
     RenderPrimitive* vao = p ? const_cast<RenderPrimitive *>(p) : &mDefaultVAO;
     update_state(state.vao.p, vao, [&]() {
         glBindVertexArray(vao->vao);
-        // update GL_ELEMENT_ARRAY_BUFFER, which is updated by glBindVertexArray
-        size_t targetIndex = getIndexForBufferTarget(GL_ELEMENT_ARRAY_BUFFER);
-        state.buffers.genericBinding[targetIndex] = vao->elementArray;
-        if (UTILS_UNLIKELY(bugs.vao_doesnt_store_element_array_buffer_binding)) {
-            // This shouldn't be needed, but it looks like some drivers don't do the implicit
-            // glBindBuffer().
-            glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, vao->elementArray);
-        }
```

Comment:
(If my previous comment is correct, this part also has to be kept)

Reply:
As discussed: [9c36b58](https://github.com/shapr3d/filament/pull/94/commits/9c36b5867b132b2bfc2471332ced52acea51e74f)